docs-src/python/interrupt-a-workflow-execution.md

```diff
@@ -0,0 +1,49 @@
+---
+id: interrupt-a-workflow-execution
+title: How to interrupt a Workflow Execution
+description:
+sidebar_label: Interrupt a Workflow Execution
+tags:
+  - guide-context
+---
+
+You can interrupt a Workflow Execution in one of the following ways:
+
+- [Cancel](#cancel)
+- [Terminate](#terminate)
+
+The following are the main differences between canceling and terminating a Workflow in Temporal:
+
+##### Cancel
+
+Canceling a Workflow provides a graceful way to stop Workflow Execution.
+This action resembles sending a `SIGTERM` to a process.
+
+- The system records a `WorkflowExecutionCancelRequested` event in the Workflow History.
+- The Workflow code can handle the cancelation and execute any cleanup logic.
+- A Workflow Task gets scheduled to process the cancelation.
+- The system doesn't forcefully stop the Workflow.
+
+For more information, see [How to cancel a Workflow Execution](/python/how-to-cancel-a-workflow-in-python).
+
+##### Terminate
+
+Terminating a Workflow forcefully stops Workflow Execution.
+This action resembles killing a process.
+
+- The system records a `WorkflowExecutionTerminated` event in the Workflow History.
+- The termination forcefully and immediately stops the Workflow Execution.
```

Comment:
not sure what this means technically. what happens in server-worker comms? don't know if this is correct:
- In-progress Activity Executions are canceled.
- No events are recorded after `WorkflowExecutionTerminated`: if there's an in-progress Workflow Task that returns Commands, they will be ignored.

Reply:
That is correct. The workflow worker never learns about termination, except possibly indirectly via WFT not found when replying to an in-progress WFT.